build

```diff
@@ -8,19 +8,20 @@
 const fs = require('fs');
 const crypto = require('crypto');
 const child_process = require('child_process');
+const Config = require('./config/config');
 
 process.chdir(__dirname);
 
 if (process.argv[2] === 'full') {
 	child_process.execSync('../play.pokemonshowdown.com/build-tools/build-indexes', {stdio: 'inherit'});
 }
 
-function updateIndex() {
+function updateFiles() {
 	let indexContents = fs.readFileSync('index.template.php', {encoding: 'utf8'});
 
 	// add hashes to js and css files
 	process.stdout.write("Updating hashes... ");
-	indexContents = indexContents.replace(/(src|href)="\/(.*?)\?[a-z0-9]*?"/g, function (a, b, c) {
+	indexContents = indexContents.replace(/(src|href)="\/(.*?)\?[a-z0-9]*?"/, function (a, b, c) {
```

Comment:
Did you intentionally remove the `g` flag?

Reply:
No; I'm not sure how that happened — good catch!